docs/src/run-your-own.md

```diff
@@ -163,12 +166,13 @@ hosted version, so you might not need all the listed commands._
 [3]: https://www.algolia.com/doc/guides/security/api-keys/#acl
 [4]: https://github.com/stedolan/jq/wiki/Installation
 [5]: ./config-file.html
-[6]: https://scrapy.org/
-[7]: https://github.com/algolia/docsearch-scraper
+[6]: #integration
+[7]: https://scrapy.org/
 [8]: https://github.com/pypa/pipenv
-[9]: https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv
-[10]: ./config-file.html#js_render-optional
-[11]: http://chromedriver.chromium.org/getting-started
-[12]: https://github.com/algolia/docsearch-configs/tree/master/configs
-[13]: ./assets/playground.png
-[14]: ./dropdown.html
+[9]: https://github.com/algolia/docsearch-scraper
```

Comment:
For the next time: _unless you are using some tool to do it for you_, please stop shifting all these references manually:
- it is unnecessary: references have an index, notably to support text reordering, there is no technical need to have them ordered by order of first appearance in the text
- it is error prone;
- it adds ~noise to the commit (see it this way: I am forced to double-check you didn't forget any reference update in the text)

Reply:
Sure, I thought it was more clear to have it